clients/src/main/java/org/apache/kafka/common/requests/ConsumerGroupDescribeRequest.java

```diff
@@ -94,6 +96,7 @@ public static List<ConsumerGroupDescribeResponseData.DescribedGroup> getErrorDes
             .map(groupId -> new ConsumerGroupDescribeResponseData.DescribedGroup()
                 .setGroupId(groupId)
                 .setErrorCode(error.code())
+                .setErrorMessage(error.message())
```

Comment:
Given how `getErrorDescribedGroupList` is used, I wonder if we should rather pass the message as an argument. What do you think?

Reply:
+1